overrides/values-trustee.yaml

```diff
@@ -0,0 +1,24 @@
+# Override the default values for the trustee chart
+# This lists the secret resources that are uploaded to your chosen ESO backend (default: Vault).
+# It does not contain the secrets themselves, only references to Vault paths.
+#
+# NOTE: When adding new CoCo workloads to coco.workloads, you must also add
+# corresponding spire-cert-{workload} and spire-key-{workload} entries here
+
+# Secret store configuration for External Secrets Operator
+# Points to the ClusterSecretStore that knows how to connect to Vault
+secretStore:
+  name: vault-backend
+  kind: ClusterSecretStore
+
+kbs:
+  secretResources:
+    - name: "kbsres1"
```

Comment:
Aligning with the fake status secret makes sense. Also from looking at hello-coco it's not using KBS res 1 or similar.